Allow php7 for Memcache adapter

t1gor opened the pull request:
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes/no
| License       | MIT
| Doc PR        | Add support for php7

I was surprised to see that it's not here yet. Assuming that the main repo has [php7 support already](https://github.com/php-cache/cache/blob/master/composer.json#L24), proposing to sync the change.

t1gor commented:
We're currently in the middle of migration, so I would really appreciate if you could accept and merge + release a new version (if possible). Thanks for understanding.

Nyholm commented:
Memcache has been excluded from our build because last time we revisited this there were no Memcache extension for PHP7.

Please remove the following (https://github.com/php-cache/cache/blob/master/.travis.yml#L34) and make sure the Travis build are green. Then I would be happy to merge this.

prisis commented:
@t1gor Hey :), can you merge the latest changes in you branche and remove the exclude from travis, thx

t1gor commented:
@prisis I rebased from latest master and added the suggested change. I'm not sure it's done correctly as PR's are always confusing me :) could you please take a look?

Nyholm commented on .travis.yml:
```diff
@@ -31,7 +31,6 @@ matrix:
     fast_finish: true
     exclude:
         - php: 7.0
```
Before we excluded the memcahed adapter on php 7.0. Now we exclude php 7.0.  😕 

You should remove the following as well: 

```yaml
exclude:
  - php: 7.0
```

Nyholm commented:
With the latest updates from @prisis it should work. Lets see =)

prisis commented:
@Nyholm maybe you can merge this and remove it your self, is faster

Nyholm commented:
Closed by #83
